src/py/pyodide/_util.py

```diff
@@ -0,0 +1,19 @@
+from tempfile import NamedTemporaryFile
+import shutil
+
+
+def get_format(format):
```

Comment:
I suggest checking for `.whl` and variations and returning `'zip'` for that.

Reply:
Let's just add them to `shutil` formats on initialization.

Reply:
> `.whl` and variations

Is there a place I can find the list of standard variations?

Reply:
There is `register_unpack_format` but honestly it looks easier to just do it manually in your function.

No idea about standards, I was just thinking about `whl` and `wheel` with an optional `.` prefix.

Reply:
I can just take the code from here:
https://bugs.python.org/issue39846